### DISCUSSION TITLE
Cache derivations results

### DISCUSSION THREAD

**Discussion by SrEstegosaurio** [YaLTeR/niri#134] [Ideas]
As the project is using nix it would be cool if it also provided a cache to avoid building it locally.
This could be done using something like [cachix](https://www.cachix.org/) since they do have a free plan.

Thanks for your time and the project.

**Comment by YaLTeR** [comment]
Hey, I don't use nix myself so someone who does will need to figure it out.

As a sidenote, it would be cool to have the dev flake built and checked on CI (as allowed-fail though), but only as long as it doesn't significantly increase the build time. I suspect that will require some sort of caching.

**Comment by KiaraGrouwstra** [comment]
also see https://github.com/sodiboo/niri-flake/issues/7
